Tests/ProjectSpecTests/InvalidConfigsFormatTests.swift

```diff
@@ -0,0 +1,44 @@
+import ProjectSpec
+import Testing
+import TestSupport
+import PathKit
+
+struct invalidConfigsMappingFormatTests {
```

Comment:
I wonder it's better to test using `settingsGroup` https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#setting-groups